src/Geolocator.Plugin.WindowsPhone8/GeolocatorImplementation.cs

```diff
@@ -113,6 +113,26 @@ public Task<Position> GetPositionAsync(TimeSpan? timeout, CancellationToken? can
         }
 
         /// <summary>
+        /// Gets position async and reverse geocode
+        /// </summary>
+        /// <returns>Address of the current position</returns>
+        public Task<Address> ReverseGeocodeCurrentLocation()
+        {
+            throw new NotImplementedException();
```

Comment:
Should be able to utilize: https://github.com/xamarin/Xamarin.Forms/blob/master/Xamarin.Forms.Maps.WP8/GeocoderBackend.cs